src/lib/Toaster.svelte

```diff
@@ -235,7 +237,23 @@
 						classes={toastOptions.classes || {}}
 						duration={toastOptions?.duration ?? duration}
 						unstyled={toastOptions.unstyled || false}
-					/>
+					>
+						<slot name="loading-icon" slot="loading-icon">
+							<Loader visible={toast.type === 'loading'} />
+						</slot>
+						<slot name="success-icon" slot="success-icon">
+							<Icon type="success" />
+						</slot>
+						<slot name="error-icon" slot="error-icon">
+							<Icon type="error" />
+						</slot>
+						<slot name="warning-icon" slot="warning-icon">
+							<Icon type="warning" />
+						</slot>
+						<slot name="info-icon" slot="info-icon">
+							<Icon type="info" />
+						</slot>
+					</Toast>
```

Comment:
If only we could have dynamic slot names in Svelte. 

[This will be possible using snippets in Svelte 5.](https://svelte-5-preview.vercel.app/#H4sIAAAAAAAACo1Ry07DMBD8lZVBcitFiYBbSKpy54B6bXpI0w1YOLZlbxCVlX9HdkIgvMRt1zuz65nxrBUSHcv3nqm6Q5azO2NYwuhsQuNeUBKyhDnd2ya8FK6xwtCmUhWJzmhLcC8cQWt1BzzNQpOONH4bQBIJBGHnoIRLRzXhah_eK_IQruTAm96R7ngCz3jOgbda8wQIXykHHqhXfEj-phxru6Rcf6cobbtaLmE3fAiow7pSRfahTBVRk4__HqJWf-GUMAYJWq1XYbAexgM7VCe0Qj2GychJw4k49tlEG5ZLjrX9Zcmxtn8tKaLDG5awTp9EK_DEcrI9DsmcYAD8N8KQjh_zSSBNUyc1PVhtHAwhLxPK1fr2qz29nEzBunma4q1dLCZB_kK0MMo4G4SyLOfQJkRFftto5Qjej0I5l24fuc94PszwQoqN39ro1AycXCwyKTbT5RylwwVrh4-9XPr6mZCJdvI5yIkm9_IHiw_DGxgyf_gwAwAA)